ziti/client.go

```diff
@@ -333,8 +338,9 @@ func (self *CtrlClient) GetServices() ([]*rest_model.ServiceDetail, error) {
 	for {
 		params.Limit = &pageLimit
 		params.Offset = &pageOffset
-		params.ConfigTypes = self.ConfigTypes
-
+		if self.supportsSetOfConfigTypesOnServiceList() {
```

Comment:
silly nit, but why are the Limit/Offset/ConfigTypes set in the loop. they never change, right? why not pull them outside the loop?

Reply:
offset gets changed, but the other two don't. moved those out of the loop